pkg/cloudprovider/backup_service.go

```diff
@@ -63,9 +67,10 @@ type BackupGetter interface {
 }
 
 const (
-	metadataFileFormatString = "%s/ark-backup.json"
-	backupFileFormatString   = "%s/%s-data.tar.gz"
-	logFileFormatString      = "%s/%s-logs.gz"
+	metadataFileFormatString   = "%s/ark-backup.json"
+	backupFileFormatString     = "%s/%s-data.tar.gz"
```

Comment:
do you want to change this back to the prior format in this PR?

Reply:
Will do tomorrow.